From: Jurica Stanojkovic <Jurica.Stanojkovic@imgtec.com>
Date: Wed, 29 Oct 2014 13:55:56 +0000
Subject: FTBFS on many architectures, test suite errors

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=750857

Use ip6-loopback for TEST_IP and increment the timeout values so the tests
won't fail on slow build machines.
---
 t/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/Makefile b/t/Makefile
index c7e7e2c..4c39bac 100644
--- a/t/Makefile
+++ b/t/Makefile
@@ -6,13 +6,13 @@
 # simple configuration choices.
 
 export TEST_HOST ?= localhost
-export TEST_IP ?= ::1
+export TEST_IP ?= ip6-loopback
 # chosen at random:
 export TEST_PORT ?= 9932
 export MSVA_PORT ?= 9933
 
-export TEST_GAP ?= 1.5
-export TEST_QUERY_DELAY ?= 2
+export TEST_GAP ?= 4
+export TEST_QUERY_DELAY ?= 5
 
 all: setup.done
 	./runtests
